app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/PlanWizardVMStep.js

```diff
@@ -49,6 +59,26 @@ class PlanWizardVMStep extends React.Component {
     } = this.props;
     const discoveryMode = vm_choice_radio === 'vms_via_discovery';
 
+    const overwriteCsvConfirmModal = (
```

Comment:
Can we please move this out of the `render` method and make it an instance method?  This variable is being set every time this component re-renders

You can see this by placing a break point on line 63 and clicking on the select all checkbox

Reply:
Actually, nvm.  It actually gets called more often as an instance method.  My bad.  As you were...